webpack/assets/javascripts/react_app/common/Tour/Tour.js

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import Tour from 'reactour';
+import { noop } from '../helpers';
+import TourButton from './TourButton';
+
+const FOREMAN_COLOR = '#00759A';
+
+export const wrapComponentWithTour = (WrappedComponent, steps, id) => {
+  const tour = class TourComponent extends Component {
```

Comment:
Does it make sense to you to move the class to a separate file?

Then you can have a class component that receives `children`, `steps` and `id`. 
And a very simple HOC that renders that component with the `WrappedComponent` as children.